test/parallel/test-quic-quicsocket.js

```diff
@@ -136,3 +136,42 @@ socket.on('ready', common.mustCall(() => {
   socket.destroy();
   assert(socket.destroyed);
 }));
+
+socket.on('close', common.mustCall(() => {
+  const err = { code: 'ERR_QUICSOCKET_DESTROYED' };
+  const message = (method) => `Cannot call ${method} ` +
+                              'after a QuicSocket has been destroyed';
+
+  assert.throws(
+    () => socket.setTTL(1),
+    { ...err, message: message('setTTL') }
```

Comment:
Nit: perhaps this would be simpler and avoids duplication
```javascript
const makeError = (method) => ({
  code: 'ERR_QUICSOCKET_DESTROYED',
  message: `Cannot call ${method} after a QuicSocket has been destroyed`;
});

// and then
  assert.throws(
    () => socket.setMulticastTTL(1),
    makeError('setMulticastTTL'),
  );

// or probably
assert.throws(() => socket.setMulticastTTL(1), makeError('setMulticastTTL'));
```

Reply:
Yeap! Give me few hours to follow that suggestion